server/controllers/logstreaming_controller.go

```diff
@@ -111,14 +114,17 @@ func (j *JobsController) GetProjectJobs(w http.ResponseWriter, r *http.Request)
 }
 
 func (j *JobsController) GetProjectJobsWS(w http.ResponseWriter, r *http.Request) {
+	jobsMetric := j.StatsScope.Scope("get_project_jobs_ws")
 	projectInfo, err := newProjectInfo(r)
 	if err != nil {
+		jobsMetric.Scope("project_info").NewCounter(metrics.ExecutionErrorMetric).Inc()
```

Comment:
Typically you don't send stats for individual errors. Only for an error aggregate.  Stats are used for alerting while logs are used to debug the alert and root cause it. Let's remove all these.